.github/workflows/release.yaml

```
@@ -49,11 +49,21 @@ jobs:
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
+      - name: Check changed version
+        id: file-changed
+        run: |
+          shopt -s nullglob
+          # Get packed chart file name
+          PKG_NAME=(.cr-release-packages/*.tgz)
+          if [ ${#PKG_NAME[@]} -gt 0 ]; then
+            echo "::set-output name=changed::true"
+          fi
+
       - name: Push Helm Chart to OCI Registries
+        if: steps.list-changed.outputs.changed == 'true'
```

Should it be `file-changed`?

```suggestion
        if: steps.file-changed.outputs.changed == 'true'
```